config/spotbugs/exclude.xml

```diff
@@ -277,4 +277,10 @@
         <Bug pattern="NP_NONNULL_PARAM_VIOLATION"/>
     </Match>
 
+    <!-- Null return is used to denote method completion with a Void return type -->
+    <Match>
+        <Class name="com.mongodb.internal.operation.AbstractWriteSearchIndexOperation"/>
+        <Method name="execute"/>
+        <Bug pattern="NP_NONNULL_RETURN_VIOLATION"/>
+    </Match>
```

Comment:
I can build and generate javadocs (`./gradlew clean check jar testClasses docs -x test -x integrationTest`) without this change. Weirdly, Evergreen reports `evergreen/static-checks` failing even with the change.

Do you actually need this change locally to make build pass?

Reply:
This was an unintentional oversight after reverting rewritable writes. I've now reverted this change too. Thanks for pointing it out!